tools/print-evaluated-rules.js

```diff
@@ -10,6 +11,12 @@ const { ESLint } = require('eslint');
     const configTypescript = await eslint.calculateConfigForFile(`${__dirname}/../test/typescript/index.js`);
     const configTypescriptTypechecked = await eslint.calculateConfigForFile(`${__dirname}/../test/typescript-type-checking/index.js`);
     const configAngular = await eslint.calculateConfigForFile(`${__dirname}/../test/angular/index.ts`);
+    const configAngularExtra = Object.keys(angularExtra.rules)
```

Comment:
This variable doesn't represent the evaluated config like the others. Maybe a name like `angularExtraRulesNotOverridden`?